### PR TITLE
Adds Mutone, Plexium, Venenum

### PR DESCRIPTION
**Pull request by BurgerLUA**
Adds three new chemicals. View the feedback thread for more information.

https://forums.aurorastation.org/viewtopic.php?f=21&t=12218

### CODE REVIEW

**Comment by BurgerLUA** [comment]
!review

**Comment by BurgerLUA** [comment]
Got everything to work. I also added Venenum as an uplink item.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm [hunk 1]
```diff
@@ -471,13 +478,101 @@
...
-/datum/reagent/luminol/touch_obj(var/obj/O)
-	O.reveal_blood()
+/datum/reagent/mutone
+	name = "Mutone"
+	id = "mutone"
+	description = "A strange green powder with even stranger properties."
+	reagent_state = SOLID
+	color = "#11AA11"
+	metabolism = (5/60) //5u every 60 seconds.
+	taste_description = "a slot machine"
```
Comment (Arrow768) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm]: How does a slot machine taste ?

Comment (BurgerLUA) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm]: have you ever licked a slot machine?

**Pull request merged**